### PR TITLE
Add subject: Physical Education

### PR DESCRIPTION
**Pull request by frankieroberto**
This is a common degree subject, particular among teacher training applicants, having been entered 419 times by candidates.

Examples of degree courses:

* https://www.uea.ac.uk/course/undergraduate/bsc-physical-education/2023
* https://www.leedsbeckett.ac.uk/courses/physical-education-ba/
* https://www.brighton.ac.uk/courses/study/physical-education-ba-hons.aspx

It does not currently have a HESA code. However the HESA subject codes for ITT use the [HECoS subject vocabulary](https://www.hesa.ac.uk/support/documentation/hecos) and the documentation for `100433` (sport and exercise sciences) states "Includes non-teacher training physical education."

### CODE REVIEW

File: lib/dfe/reference_data/degrees.rb [hunk 1]
```diff
@@ -7145,6 +7145,12 @@ module Degrees
             synonyms: [],
             dttp_id: '4d8370f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '100750' },
+          'ba1ffe89-73fb-4de8-a48d-12e644ee5687' =>
+          { name: 'Physical Education',
```
Comment (edwardhorsford) [inline lib/dfe/reference_data/degrees.rb:7149]:
```suggestion
          { name: 'Physical education',
```

File: lib/dfe/reference_data/degrees.rb [hunk 2]
```diff
@@ -7145,6 +7145,12 @@ module Degrees
...
+            synonyms: ['PE'],
+            dttp_id: nil,
+            hesa_itt_code: '100433',
```
Comment (edwardhorsford) [inline lib/dfe/reference_data/degrees.rb:7152]: I'm not sure we should map this...

**Comment by edwardhorsford** [comment]
Our team has put in a request to get this subject added to HECOS. In the mean time I'm unsure if we should map to this HECOS code.

**Review by edwardhorsford — CHANGES_REQUESTED** [review]

File: lib/dfe/reference_data/degrees.rb (hunk 1 above)
Comment (frankieroberto) [reply lib/dfe/reference_data/degrees.rb:7149]: 🤔

File: lib/dfe/reference_data/degrees.rb (hunk 2 above)
Comment (frankieroberto) [reply lib/dfe/reference_data/degrees.rb:7152]: It seems correct _for now_, at least according to the comment on the HECoS guidance? But equally could be nil, which is what would currently happen (although then do ITT providers have to manually look up the code when reporting it to HESA?)

**Review by alaric-dfe — APPROVED** [review]

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**